python/lsst/ts/MTMount/mtmount_csc.py

```diff
@@ -357,19 +361,19 @@ async def enable_devices(self):
         self.enabled_state = enums.EnabledState.ENABLING
         try:
             enable_commands = [
-                commands.TopEndChillerResetAlarm(),
-                commands.MainPowerSupplyResetAlarm(),
-                commands.MirrorCoverLocksResetAlarm(),
-                commands.MirrorCoversResetAlarm(),
-                commands.AzimuthAxisResetAlarm(),
-                commands.ElevationAxisResetAlarm(),
+                # commands.TopEndChillerResetAlarm(),
+                # commands.MainPowerSupplyResetAlarm(),
+                # commands.MirrorCoverLocksResetAlarm(),
+                # commands.MirrorCoversResetAlarm(),
+                # commands.AzimuthAxisResetAlarm(),
+                # commands.ElevationAxisResetAlarm(),
                 commands.CameraCableWrapResetAlarm(),
-                commands.TopEndChillerPower(on=True),
-                commands.TopEndChillerTrackAmbient(on=True, temperature=0),
-                commands.MainPowerSupplyPower(on=True),
-                commands.OilSupplySystemPower(on=True),
-                commands.AzimuthAxisPower(on=True),
-                commands.ElevationAxisPower(on=True),
+                # commands.TopEndChillerPower(on=True),
+                # commands.TopEndChillerTrackAmbient(on=True, temperature=0),
+                # commands.MainPowerSupplyPower(on=True),
+                # commands.OilSupplySystemPower(on=True),
+                # commands.AzimuthAxisPower(on=True),
+                # commands.ElevationAxisPower(on=True),
                 commands.CameraCableWrapPower(on=True),
                 commands.CameraCableWrapEnableTracking(on=True),
```

Comment:
Note for the actual CSC, I don't think we want to switch on `commands.CameraCableWrapEnableTracking`. From what I understood this is the mode where CCW will track rotator automatically.